tests/cli/test_cli_magic.py

```diff
@@ -48,13 +48,29 @@ def parse_cli_string(self, cli_words: str):
         for key, value in zip(dict_keys, dict_values):
             value = value.strip('\"')
             value = value.strip('\'')
-            self.cfg[key] = value
+            try:
+                value = int(value)
+            except ValueError:
+                pass
+
+            key_subparts = key.split('.')
+            if len(key_subparts) == 1:
+                self.cfg[key] = value
+            elif len(key_subparts) == 2:
+                self.cfg[key_subparts[0]][key_subparts[1]] = value
+            else:
+                raise ValueError(
+                    f"Keys with more than 2 subparts are not supported,"
```

Comment:
Above you used single quotes and here you use double quotes. I would always use single quotes.